src/features/metadata-instance-editor/Instance.js

```diff
@@ -549,10 +551,10 @@ class Instance extends React.PureComponent<Props, State> {
         return this.canEdit() && isEditing;
     }
 
-    renderEditButton = () => {
+    renderEditButton = (canEditInstance: boolean) => {
```

Comment:
should the boolean checks in `canEditInstance` be moved to the methods `canEdit`?